actions/build-on-commit

```diff
@@ -48,17 +66,32 @@ def add_job():
            "{{series}}": hookenv.action_get("series") or "",
            "{{controller}}": hookenv.action_get("controller") or "",
            "{{charm_home}}": getcwd(),
+           "{{trigger}}": trigger,
+           "{{skip_builds}}": skip_builds,
            }
 
     template_path_source = "templates/BuildMyCharm/config.xml"
+    job_name = "charm-{}".format(charm_fname)
     with open(template_path_source) as infile:
         data = infile.read()
         for src, target in rep.items():
             data = data.replace(src, target)
         try:
-            jclient.create_job("charm-{}".format(charm_fname), data)
+            jclient.create_job(job_name, data)
         except JenkinsException as e:
             cwrhelpers.fail_action(str(e))
+    token = get_hook_token(job_name)
+    url = "http://<Jenkins_machine>:{}{}/trigger/{}/{}".format(REST_PORT,
+                                                         get_rest_path(),
+                                                         job_name,
+                                                         token)
```

Comment:
Nitpick: this would be more user friendly if instead of <Jenkins machine>, we included the IP address, which I think that we can get out of the hookenv.

Of course, that's only useful if the IP address is publicly exposed, and you probably want to have setup DNS for something like jenkins.mycompany.com. It still feels more friendly to start people out with that Ip address ...